pymc_marketing/mmm/plot.py

```diff
@@ -262,6 +262,12 @@ def plot_curve(
         Figure and the axes
 
     """
+    scalar_coords_to_drop = []
+    for coord, values in curve.coords.items():
+        if values.size == 1 and coord not in curve.indexes:
+            scalar_coords_to_drop.append(coord)
+
+    curve = curve.reset_coords(scalar_coords_to_drop, drop=True)
```

Comment:
On second thought, can you wrap this whole thing in a function? say, `drop_scalar_coords`

This would want to be called for not just `plot_curve` but also `plot_samples` and `plot_hdi`

Pulling out as a function will will help with testing

Reply:
sure, will put it on utils